### DISCUSSION TITLE
nothrow keyword for catching all exceptions

### DISCUSSION THREAD

**Discussion by Xyncgas** [dotnet/csharplang#5800] [Language Ideas] [answered]
```C#
nothrow MyMethodA();
nothrow
{
    MyMethodB();
}
```
Is basically :
```C#
try{MyMethodA();}catch{}
try
{
    MyMethodB();
}
catch
{
}
````

**Comment by spydacarnage** [comment]
That's not _catching_ any exceptions - it's ignoring them, which I don't think is something the language would ever want to make _easier_ to do.

**Reply by Xyncgas** [reply]
> don't think is something the language would ever want to make easier to do.

Thanks for replying, I am a programmer I choose it to nothrow and I will get blamed for it, this is something I am going to do anyways for good reasons

**Comment by theunrepentantgeek** [comment, marked as answer]
Dupe of rejected issue #220 where the [top voted answer](https://github.com/dotnet/csharplang/issues/220#issuecomment-283895349) is:

> `catch {}` is evil. There are times when it might make sense to use it after careful analysis of the consequences. 
> Having the compiler encourage its casual use through a syntactic shortcut though, would be a seriously bad thing to do.
